pom.xml

```diff
@@ -16,64 +16,89 @@
-        <configuration>
-          <!-- javadoc:jar uses outputDirectory - but disabled since it doesn't create the jar for some reason -->
-          <!-- outputDirectory>${project.reporting.outputDirectory}/../../../www/2.1-SNAPSHOT</outputDirectory -->
-          <!-- javadoc:javadoc uses reportOutputDirectory -->
-          <reportOutputDirectory>${project.reporting.outputDirectory}/../../../www/2.1-SNAPSHOT</reportOutputDirectory>
-          <destDir>apidocs</destDir>
-        </configuration>
-      </plugin>      
-    </plugins>     
-  </build>
-  
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd"> <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.eclipse.ee4j</groupId>
+        <artifactId>project</artifactId>
+        <version>1.0.1</version>
+    </parent>
+
+    <groupId>jakarta.jms</groupId>
+    <artifactId>jakarta.jms-api</artifactId>
+    <version>2.1-SNAPSHOT</version>
+    
+    <name>jms</name>
+    <url>https://projects.eclipse.org/projects/ee4j.jms</url>
+
+    <build>
+        <plugins>
+           <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>2.3.2</version>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                </configuration>
+            </plugin>
+            
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-source-plugin</artifactId>
+                <version>3.0.1</version>
+                <configuration>
+                    <includePom>true</includePom>
+                </configuration>
+                <executions>
+                    <execution>
+                       <id>attach-sources</id>
+                       <goals>
+                           <goal>jar-no-fork</goal> 
+                       </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <version>2.8</version>
+                <configuration>
+                    <groups>
+                        <group>
+                            <title>JMS API Documentation</title>
+                            <packages>
+                                javax.jms
+                            </packages>
+                        </group>
+                    </groups>
+                
+                    <!-- Specifies the text to be placed at the bottom of each output file. -->
+                    <bottom>Copyright 1997-2018 Oracle America Inc. All rights reserved. Use is subject to &lt;a
```

Comment:
Should this be changed to Eclipse Foundation?

Reply:
Not sure, I don't think any of the other projects have changed it.

Reply:
Let's see if anyone else have comments about it. We're in this limbo between licenses, but it should probably have the Oracle notice until the agreement has been signed.

Reply:
Okay. Regardless, we can always change it later if needed (but then it probably should be a PMC recommendation or so, so that not a single project changes it, but all of them do in the same way)

Reply:
I agree. I'll approve :)

Reply:
Oracle retains the copyright; Oracle is not assigning the copyright to the Eclipse Foundation.